command/testing.go

```diff
@@ -2,11 +2,103 @@ package command
 
 import (
 	"errors"
+	"fmt"
+	"os/exec"
+	"reflect"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/core"
 
 	"github.com/cli/cli/api"
 	"github.com/cli/cli/context"
+	"github.com/cli/cli/test"
+	"github.com/cli/cli/utils"
 )
 
+// TODO this is split between here and test/helpers.go. I did that because otherwise our test
+// package would have to import utils (for utils.Runnable) which felt wrong. while utils_test
+// currently doesn't import test helpers, i don't see why that ought to be precluded.
```

Comment:
Good thinking. I'm definitely up for extracting parts of utils into isolated packages, or even interface-only packages 👍

Reply:
cool--i definitely felt a desire for interface-only packages while working on this stuff.